### DISCUSSION TITLE
Add rows to a table form

### DISCUSSION THREAD

**Discussion by rmutili** [Budibase/budibase#2081] [Ideas]
Hi,

Is there a way to create a table in a form which allows users to add dynamic rows and fill them in and then submit? I am working on an expense request system and it supports different adding rows to a table form.

**Comment by Bogey** [comment]
I have the same situation. I haven’t tried it yet but perhaps this is how: [https://github.com/Budibase/budibase/discussions/2043](https://github.com/Budibase/budibase/discussions/2043 )
What do you think?

**Reply by rmutili** [reply]
Seems like the same functionality but in this case, I want a whole row... And possibilitty to add many rows and submit..
